lib/stackup/parameters.rb

```diff
@@ -75,7 +75,26 @@ def value
       if use_previous_value
         :use_previous_value
       else
-        parameter_value
+        ParameterValue.new(parameter_value).to_param
+      end
+    end
+
+  end
+
+  class ParameterValue
+
+    def initialize(value)
+      @value = value
+    end
+
+    def to_param
+      case @value
+      when TrueClass, FalseClass, Integer then
+        @value.to_s
+      when Array
+        @value.map { |v| ParameterValue.new(v).to_param }.join(",")
+      else
+        @value
```

Comment:
I'm curious why this is a class. Couldn't it just be a plain function?

Reply:
> I'm curious why this is a class. Couldn't it just be a plain function?

Was following the convention used elsewhere in the same file, e.g. `ParameterStruct`